test/framework/src/main/java/org/elasticsearch/test/discovery/TestZenDiscovery.java

```diff
@@ -67,7 +67,7 @@
         Setting.boolSetting("discovery.zen.use_zen2", false, Setting.Property.NodeScope);
 
     public static final Setting<Boolean> USE_ZEN2_PERSISTED_STATE =
```

Comment:
can we remove this now?

Reply:
Yes, sure. I just wanted to keep it if at some point we want for some reason to run integration tests with `InMemoryPersistedState`, but I can hardly see a reason to do so. https://github.com/elastic/elasticsearch/pull/36013/commits/745004213059e36bdf81135df7911d5a19bc36cf